app/views/notifier/review_orders.text.erb

```diff
@@ -1,3 +1,3 @@
-<%= text(".body", facility: @facility.to_s, account: @account.to_s) %>
+<%= text(".body", facility: @facility.to_s, accounts: @accounts.html_safe) %>
```

Comment:
I was seeing entity encoding here, which we don't want in plain text. That said, I'm thinking I should have wrapped it in `raw` instead. Updating.